Base.rte/GUIs/MultiplayerGameGUI.ini

```diff
@@ -9,6 +9,7 @@ Visible = True
 Enabled = True
 Name = base
 Anchor = Left, Top
+ToolTip = 
```

Comment:
Add `None` to empty tooltips.
I'm pretty sure I added a garbage fix to deal with them sometimes being shitty but better be safe (even though they don't show here but whatever. Consistency:tm:).

Reply:
Will do. Does gui maker not spit out None?